setup.cfg

```diff
@@ -0,0 +1,116 @@
+# This file is used to configure your project.
+# Read more about the various options under:
+# http://setuptools.readthedocs.io/en/latest/setuptools.html#configuring-setup-using-setup-cfg-files
+
+[metadata]
+name = hmf
+description = A halo mass function calculator
+author = Steven Murray
+author-email = steven.g.murray@asu.edu
+license = mit
+long-description = file: README.rst
+long-description-content-type = text/x-rst; charset=UTF-8
```

Comment:
This attr doesn't have a kebab-case alias like some of the others: https://github.com/steven-murray/hmf/commit/ee4b1d7300e096c34feae2dd61d8c81bc6a9d18b#r39336456 / https://setuptools.readthedocs.io/en/latest/setuptools.html#metadata.